binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -153,6 +163,13 @@ public void run() {
             }
         }
 
+        private void setThingProperty(Thing thing, String key, String value) {
```

Comment:
Would it make sense to move this to the BaseThingHandler as a convenience method?
And is the thing really required as a parameter? I would rather remove it and call getThing() inside the method to get hold of it.